clubs/management/commands/find_broken_images.py

```diff
@@ -1,28 +1,36 @@
+import os
+
 import requests
+from django.conf import settings
 from django.core.management.base import BaseCommand
 
 from clubs.models import Club
 
 
 class Command(BaseCommand):
-    help = 'List clubs with broken images.'
+    help = 'List clubs with broken images and delete the image link.'
 
     def handle(self, *args, **kwargs):
         broken_list = []
         working_list = []
-        for club in Club.objects.all():
+        for club in Club.objects.filter(image__isnull=False):
             if club.image:
-                resp = requests.head(club.image.url)
-                if not resp.ok:
-                    self.stdout.write(self.style.ERROR('{} has broken image {}'.format(club.id, club.image.url)))
+                image_ok = False
+                if club.image.url.startswith('http'):
```

Comment:
Nit: if a club has an image served over http and not https it is a security vulnerability and the browser will raise some complaints with the https cert for the website. Any way we could make them swap over to https?

Also, depending on how many images are originally defined but then down the line the URL no longer points to the asset, do we think it could be worth moving to storing images in like S3? Obv not something we'd tackle in this PR

Reply:
We actually should be fine with the http/https thing, since we already self-host all of their images on S3 already. The only case where the image URL is not hosted/controlled by us was during the Groups Online @ Penn import script.

The check in the code is more for local development environment vs production environment, in the local environment you can't properly do a `requests.head` because the server isn't running in the management script.